.github/workflows/deploy-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Deploy tests
+
+on:
+  pull_request:
+
+jobs:
+  assess-file-changes:
+    uses: catalystneuro/neuroconv/.github/workflows/assess-file-changes.yml@disable_tests_if_no_changes
```

Comment:
What is the @disable_tests_if_no_changes doing in this section? Is it referencing this branch? Why do you need to reference the files in such an absolute way? Interesting

Reply:
B/c the workflow only exists on this branch at the moment 😆 would have to update that to main after merge

Reply:
Yes, I guess it is because the action machinery sits outside the repo and then you can't just call relative scripts.